src/api/app/models/comment.rb

```diff
@@ -27,6 +21,20 @@ def to_s
   def create_notification(params = {})
     params[:commenter] = user.id
     params[:comment_body] = body
+    params[:commenters] = involved_users((object_type.underscore + '_id').to_sym, object.id)
+
+    case object_type
+    when "Package"
+      params[:package] = object.name
+      params[:project] = object.project.name
+    when "Project"
+      params[:project] = object.name
+    when "BsRequest"
+      params = object.notify_parameters(params)
+    end
+
+    # call the action
+    Event::CommentForPackage.create params
```

Comment:
In the child models we called three different actions:

- Event::CommentForPackage.create
- Event::CommentForProject.create
- Event::CommentForRequest.create

Doesn't that matter anymore?